bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/actions/AVMFritzHeatingActions.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.avmfritz.internal.actions;
```

Comment:
I am actually not certain myself anymore, but shouldn't actions be in exported packages, so that they can be seen by the rule engine? Scanning our repo here, I noticed that we have a few bindings putting actions in exported packages, while others have it in internal packages...

Reply:
Me neither. But does the rule engine really need to know them? I don't think so because we use a wrapper action to access all available `ThingActions` by filtering for scope and `ThingUID`.

Reply:
It is using the wrapper at runtime, but for the DSL and specifically the content completion in the editor, it is imho required:

<img width="820" alt="Bildschirmfoto 2020-08-05 um 22 50 37" src="https://user-images.githubusercontent.com/3244965/89462755-2e37ba80-d76e-11ea-8942-bd1c21ce6b1b.png">

The mail action is so far the only one I am using, but that one is in an exported package, so I'd suggest to do the same with the others.

Reply:
Good point. I moved them.

Reply:
I see that you moved not only the Actions class, but also the `AVMFritzHeatingActionsHandler` interface - is this necessary to be exposed? If not, I'd say it should stay internal.

Reply:
I moved it back to an internal package.